### PR TITLE
Added new [Term Entry] Java Calendar get()

### PR DESCRIPTION
**Pull request by AapoSoukkio**
### Description

- Added `get()` method as a new term entry for the [Calendar](https://www.codecademy.com/resources/docs/java/calendar) class.
- New file has been created in `content/java/concepts/calendar/terms/get/get.md`
- Closes Issue #2973 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/get/get.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Calendar'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.get()`** method of Calendar class is used to return the value of the given calendar field in the parameter.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/get/get.md:15]:
```suggestion
The **`.get()`** method of the `Calendar` class is used to return the value of the given calendar field.
```

File: content/java/concepts/calendar/terms/get/get.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+## Syntax
+
+The get(int field) is used to retrieve the value of a specific date or time field, indicated by the provided field parameter.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/get/get.md:19]:
```suggestion
The `get(int field)` is used to retrieve the value of a specific date or time field, indicated by the provided field argument.
```

File: content/java/concepts/calendar/terms/get/get.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+```pseudo
+public int get(int field)
+```
+
+## Example 1
+
+This example demonstrates the utilization of the Calendar class. It acquires the present year, month, and day of the month from a Calendar instance, adjusting the month representation to account for the 0-based index.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/get/get.md:27]:
```suggestion
This example demonstrates a basic implementation of the `.get` function. The code acquires the present year, month, and day of the month from a `Calendar` instance, adjusting the month representation to account for the 0-based index.
```

File: content/java/concepts/calendar/terms/get/get.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+```shell
+Current Year: 2023
+Current Month: 8
+Day of the Month: 9
+```
+
+## Example 2
+
+This example employs the Calendar class to set a fixed date (July 16, 2022) within a Calendar instance. It retrieves the year, month, and day of the month from the configured instance and displays them in a formatted output on the console.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/get/get.md:62]:
```suggestion
This example employs the `Calendar` class to set a fixed date (July 16, 2022) within a `Calendar` instance. It retrieves the year, month, and day of the month from the configured instance and displays them in a formatted output on the console.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@AapoSoukkio I've added my initial comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by AapoSoukkio** [comment]
@caupolicandiaz  Thank you for your review👍 The entries has been updated as requested now.

File: content/java/concepts/calendar/terms/get/get.md (hunk 4 above)
Comment (AapoSoukkio) [reply content/java/concepts/calendar/terms/get/get.md]: I completely overlooked the use of back sticks. Thank you for pointing that out! 🙌

**Review by SSwiniarski — APPROVED** [review]
@AapoSoukkio Looks good!

**Pull request merged**